sdk/core/azure-core/src/main/java/com/azure/core/util/BinaryData.java

```diff
@@ -134,9 +111,6 @@ public static BinaryData fromStream(InputStream inputStream) {
      * Creates an instance of {@link BinaryData} from the given {@link Flux} of {@link ByteBuffer}.
      * <p>
      * If the {@code data} is null an empty {@link BinaryData} will be returned.
-     * <p>
-     * <b>Note:</b> This will collect all bytes from the {@link ByteBuffer ByteBuffers} resulting in {@link
-     * ByteBuffer#hasRemaining() hasRemaining} to return false.
```

Comment:
It would be good in these docs to clearly call out the same thing I've mentioned above - when memory copies will be performed, etc.